Packs/Code42/Integrations/Code42/Code42_test.py

```diff
@@ -802,8 +802,8 @@
                 "filterClause": "AND",
                 "filters": [
                     {"operator": "IS_NOT", "term": "exposure", "value": "IsPublic"},
-                    {"operator": "IS_NOT", "term": "exposure", "value": "OutsideTrustedDomains"},
                     {"operator": "IS_NOT", "term": "exposure", "value": "SharedViaLink"},
+                    {"operator": "IS_NOT", "term": "exposure", "value": "OutsideTrustedDomains"},
```

Comment:
Why was this moved? I think these need to be in alphabetical order due to changes that were made that make the filters get sorted when are is called on them.

Reply:
I am not sure but why, but the tests expect a different order

Reply:
Ah, my bad, my py42 was outdated in that environment